### PR TITLE
Upgrade Glance Client to 1.1.0

### PR DESCRIPTION
**Pull request by mirrorcoder**
Fix get_ref_image()
Upgrade glance client

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -20,5 +19,6 @@ pyyaml
 redis
 requests==2.2.1
 sqlalchemy
+oslo.utils==1.9.0
```
Comment (miarmak) [inline requirements.txt]: Why do we need this package?

File: cloudferrylib/os/image/glance_image.py [hunk 2]
```diff
@@ -144,7 +144,10 @@ def get_image_status(self, image_id):
 
     def get_ref_image(self, image_id):
         try:
-            return self.glance_client.images.data(image_id)._resp
+            return getattr(self.glance_client.images.data(image_id),
+                           '_resp',
```
Comment (miarmak) [inline cloudferrylib/os/image/glance_image.py]: no way to get rid of it?

File: cloudferrylib/utils/iterator_as_file.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2014 Mirantis Inc.
```
Comment (miarmak) [inline cloudferrylib/utils/iterator_as_file.py]: 2015 :wink:

File: cloudferrylib/utils/iterator_as_file.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+from utils import get_log
+
+LOG = get_log(__name__)
```
Comment (miarmak) [inline cloudferrylib/utils/iterator_as_file.py]: We don't use this variable

File: requirements.txt (hunk 1 above)
Comment (roman-verchikov) [inline requirements.txt]: I assume it was simply moved from above ;) This change is not needed though, so I would recommend reverting it

**Comment by ntymtsiv** [comment]
-1, mirrorcoder  wants to merge 1 commit into MirantisWorkloadMobility:master
Create pull request to devel branch

**Pull request closed without merging**